libs/dotnet-sdk/README.md

```diff
@@ -0,0 +1,129 @@
+## DotNet SDK
+
+## Installation
+
+```powershell
+dotnet add package Featureboard.DotnetSdk
+```
+
+
+## Setup
+
+1) Create a Features model
+```csharp
+using Featureboard.DotnetSdk.Models;
+
+public class WeatherFeatures : IFeatures
+{
+  public bool WeatherImperial { get; set; }
+}
+```
+
+2) Implement the audance provider
+```csharp
+using Featureboard.DotnetSdk;
+
+public class QueryStringAudienceProvider : IAudienceProvider
+{
+  public List<string> AudienceKeys { get; }
+
+  public QueryStringAudienceProvider(IHttpContextAccessor contextAccessor)
+  {
+    if (contextAccessor.HttpContext?.Request.Query.TryGetValue("audience", out var audienceKeys) ?? false)
+    {
+      AudienceKeys = audienceKeys.Where(string.IsNullOrWhiteSpace).ToList()!;
+    }
+
+    AudienceKeys = new List<string>();
+  }
+}
+```
+
+
+3) Register the provider in `program.cs`
+```csharp
+// Register featureboard
+builder.Services.AddFeatureBoard<WeatherFeatures, QueryStringAudienceProvider>()
+  .WithPollingUpdateStrategy();
```

Comment:
Can we default this so if it's omitted things can still resolve? (low care)

Reply:
I can't think of a way to do this as you'd have to register in the AddFeatureBoard method and then unregister it with the second one